samples/src/main/java/okio/samples/SocksProxyServer.java

```diff
@@ -56,7 +56,11 @@
 
   public void start() throws IOException {
     serverSocket = new ServerSocket(0);
-    executor.execute(this::acceptSockets);
+    executor.execute(new Runnable() {
```

Comment:
FYI, looks like my SocksProxyServer change collided with @Egorand’s build change. For some reason Gradle didn’t like this being Java 8 with the other things as Java 7. I just downgraded this to Java 7 to unblock.

Reply:
I'll look into making it Java 8